### PR TITLE
feat: update fees

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

File: docs/releases/testnet.md [hunk 1]
```diff
@@ -47,10 +47,12 @@ This fee only depends on the source/destination chain and the asset and does NOT
 When transferring an asset X from chain Y to chain Z, the transfer fee is the sum of per-chain fee for that asset.
 For e.g. a transfer of 1000 UST from Terra to Avalanche will have a fee of 1.5 UST (= 0.5 UST for Terra + 1.0 UST for Avalanche), and so the recipient will get 998.5 UST.
 
-| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos (Terra) |
-| ------------ | -------- | ---------------- | -------------- |
-| UST          | 20 UST   | 1 UST            | 0.5 UST        |
-| LUNA         | 0.2 LUNA | 0.01 LUNA        | 0.005 LUNA     |
+| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos (Terra) | Decimals  | Unit     |
```
Comment (ggutoski) [inline docs/releases/testnet.md:50]: should we rename the `Cosmos (Terra)` column to `Cosmos`?  Or maybe `Cosmos (eg. Terra)`

**Review by ggutoski — APPROVED** [review]

File: docs/releases/testnet.md (hunk 1 above)
Comment (milapsheth) [reply docs/releases/testnet.md:50]: Some people from EVM chain land don't know about Terra. I'll modify it to have `eg.`

File: docs/releases/mainnet.md [hunk 2]
```diff
@@ -57,10 +57,13 @@ This fee only depends on the source/destination chain and the asset and does NOT
 When transferring an asset X from chain Y to chain Z, the transfer fee is the sum of per-chain fee for that asset.
 For e.g. a transfer of 1000 UST from Terra to Avalanche will have a fee of 1.5 UST (= 0.5 UST for Terra + 1.0 UST for Avalanche), and so the recipient will get 998.5 UST.
 
-| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos (Terra) |
-| ------------ | -------- | ---------------- | -------------- |
-| UST          | 20 UST   | 1 UST            | 0.5 UST        |
-| LUNA         | 0.2 LUNA | 0.01 LUNA        | 0.005 LUNA     |
+| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos (Terra) | Decimals  | Unit     |
```
Comment (milapsheth) [inline docs/releases/mainnet.md:60]:
```suggestion
| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos Chains | Decimals  | Unit     |
```

File: docs/releases/testnet.md (hunk 1 above)
Comment (milapsheth) [inline docs/releases/testnet.md:50]:
```suggestion
| Asset symbol | Ethereum | non-Ethereum EVM | Cosmos Chains  | Decimals  | Unit     |
```

Comment (milapsheth) [reply docs/releases/testnet.md]: Changed to Cosmos chains instead

**Pull request merged**